src/web3/utils/files.ts

```diff
@@ -18,3 +18,13 @@ export const downloadFile = (
   a.dispatchEvent(clickEvt)
   a.remove()
 }
+
+export async function parseJsonFile(file: File) {
```

Comment:
Non-blocking:

Maybe:
```suggestion
export async function parseJSONFile(file: File) {
```
?

Reply:
35a1cc6